test/functional/mempool_accept.py

```diff
@@ -389,6 +390,56 @@ def run_test(self):
             maxfeerate=0,
         )
 
+        self.log.info('OP_1 <0x4e73> is able to be created and spent')
```

Comment:
Could be beneficial to add tests for P2A interaction with V2/V3 transactions and RBF scenarios.  Probably best for a follow-up.

Reply:
Seems more like a (useful) demo to me. I have old branches that mashed TRUC/Ephemeral Dust/Anchors/etc together to show the use-cases, but seems out of scope for these tests.